RiotSharp/Endpoints/Interfaces/Static/IStaticDataEndpoint.cs

```diff
@@ -0,0 +1,18 @@
+﻿using RiotSharp.Endpoints.Interfaces.Static;
+
+namespace RiotSharp.Endpoints.Interfaces.Static
+{
+    public interface IStaticDataEndpoint
```

Comment:
You might rename this to `IStaticDataEndpointCollection` or `IStaticDataEndpoints`, so it's distinguishable from the actual endpoints.

Reply:
Since it is no "real" collection, the second one looks more promising.

@BenFradet what is your opinion on the change from `IStaticDataEndpoint` to IStaticDataEndpoints`?

Reply:
let's go with `IStaticDataEndpoints` 👍